CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 - build: Migrate from tslint to eslint (#113)
+- fix: Add stronger types for module exports (#114)
```

Comment:
This is different than the PR title, but figured it is the only change that could matter for a CHANGELOG.

Open to change it though.

Reply:
That's fine 👍